easybuild/framework/easyconfig/easyconfig.py

```diff
@@ -162,7 +162,7 @@ def get_toolchain_hierarchy(parent_toolchain):
         for dep in parsed_ec['dependencies']:
             ec = robot_find_easyconfig(dep['name'], det_full_ec_version(dep))
             ec = process_easyconfig(ec, validate=False)[0]
-            alldeps = [ec['ec']['toolchain']] + ec['ec']['dependencies']
+            alldeps = [ec['ec']['toolchain']] + parsed_ec['dependencies']
```

Comment:
You sure this is correct? I think this was originally correct - what was the error? I think it should probably be
```
alldeps = [ec['ec']['toolchain']] + ec['ec']['dependencies']['toolchain']
```